tests/components/datadisplay/test_datatable.py

```diff
@@ -40,13 +40,13 @@ def test_validate_data_table(data_table_state: rx.State, expected):
 
     data_table_dict = data_table_component.render()
 
-    # prefix expected with state name
-    state_name = data_table_state.get_name()
-    expected = f"{state_name}.{expected}" if expected else state_name
+    var = data_table_state
+    if expected:
+        var = getattr(var, expected)
 
     assert data_table_dict["props"] == [
-        f"columns={{{expected}.columns}}",
-        f"data={{{expected}.data}}",
+        f"columns={{{var.columns._var_name}}}",
+        f"data={{{var.data._var_name}}}",
```

Comment:
```suggestion
        f"columns={{{str(var.columns)}}}",
        f"data={{{str(var.data)}}}",
```

This should work, because `_var_name` has gone away recently as an implementation detail.